mscore/shortcut.cpp

```diff
@@ -3935,6 +3936,7 @@ static QList<Shortcut1> loadShortcuts(QString fileLocation)
       QFile f(fileLocation);
       if (!f.open(QIODevice::ReadOnly)) {
             qDebug("Cannot open shortcuts");
+            QMessageBox::critical(0, QObject::tr("ERROR"), QObject::tr("The shortcut file chosen could not load properly. Try loading another file."));
```

Comment:
`QMessageBox::critical(this, QObject::tr("Load Shortcuts"), QObject::tr("Can't load shortcuts file: ") + QString(strerror(errno)));` maybe?